dist/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Your Amazing Shows</title>
+  <script defer src="bundle3de50f2791285e4db354.js"></script></head>
+  <body>
+    <div class="container">
+      <!-- Header -->
+      <header>
+        <a href="">
+          <img
+            src="https://placehold.co/40x40/orange/white?font=Lora&text=YAS"
+            alt=""
+          >
+          <span>Your Amazing Shows</span>
+        </a>
+      </header>
+      <!-- Main-->
+      <div id="mainSection" class="main"></div>
+      <!-- Dialog -->
+      <dialog class="modal" id="modal">
+        <span class="modal-close">x</span>
+        <div class="card-image">
+          <img src="https://placehold.co/400x400/" id="card-image" alt="">
+        </div>
+        <div class="modal-body">
+          <h3 id="modal-name">name</h3>
+          <p id="modal-description"></p>
+          <h4>Comments <span id="comment-number"></span></h4>
+          <table>
+            <tbody id="table-body"></tbody>
+          </table>
+          <h4>Add a comment</h4>
+          <form id="form">
+            <div class="form-group">
+              <input
+                type="text"
+                placeholder="your name"
+                name="name"
+                id="name"
+                required
+              >
+            </div>
+            <div class="form-group">
+              <textarea
+                name="comment"
+                id="comment"
+                required
+                placeholder="Your insights"
+              >
+              </textarea>
+            </div>
+            <input type="hidden" id="item-id">
+            <button type="submit" id="submit">Comment</button>
```

Comment:
- Kindly, after adding the new comment your page is redirecting to the home page but it should update itself and the new comment should be added to the comments list :+1: so please update it as it is part of the requirement
![Screenshot from 2023-03-18 21-55-59](https://user-images.githubusercontent.com/52010789/226130358-e5a3be9d-b26d-497a-80b9-392841b3713c.png)